### PR TITLE
Set next development iteration as 1.1.0-m1-SNAPSHOT

### PR DESCRIPTION
**Pull request by renuka-fernando**
### Purpose
$subject

### Issues

Task

### Automation tests
 - Unit tests added: Yes/No
 - Integration tests added: Yes/No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: integration/test-integration/pom.xml [hunk 1]
```diff
@@ -225,7 +225,7 @@
         <dependency>
             <groupId>org.wso2.am.choreo.connect</groupId>
             <artifactId>mock-backend-server</artifactId>
-            <version>1.0.0-rc3-SNAPSHOT</version>
```
Comment (praminda) [inline integration/test-integration/pom.xml:228]: can you remove this version specification from here and use dependency management/project parent version instead?

File: integration/test-integration/pom.xml [hunk 2]
```diff
@@ -244,7 +244,7 @@
         <dependency>
             <groupId>org.wso2.am.choreo.connect</groupId>
             <artifactId>org.wso2.choreo.connect.enforcer</artifactId>
-            <version>1.0.0-rc3-SNAPSHOT</version>
+            <version>1.1.0-m1-SNAPSHOT</version>
```
Comment (praminda) [inline integration/test-integration/pom.xml:247]: same in here

File: integration/test-integration/pom.xml (hunk 1 above)
Comment (renuka-fernando) [reply integration/test-integration/pom.xml:228]: Using parent version from https://github.com/wso2/product-microgateway/pull/2586/commits/ded1c4aaff63bb7bd5eafa74e5bf2cd320106af8

File: integration/test-integration/pom.xml (hunk 2 above)
Comment (renuka-fernando) [reply integration/test-integration/pom.xml]: Using parent version from ded1c4a

**Review by praminda — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] dev-deployment : 20211216.13

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment : 20211216.15

**Pull request merged**